test/e2e/v1/e2e_suite.go

```diff
@@ -108,7 +108,8 @@ var _ = Describe("kubebuilder", func() {
 
 			// NOTE: If you want to run the test against a GKE cluster, you will need to grant yourself permission.
 			// Otherwise, you may see "... is forbidden: attempt to grant extra privileges"
-			// $ kubectl create clusterrolebinding myname-cluster-admin-binding --clusterrole=cluster-admin --user=myname@mycompany.com
+			// $ kubectl create clusterrolebinding
+			// myname-cluster-admin-binding --clusterrole=cluster-admin --user=myname@mycompany.com
```

Comment:
```suggestion
			//     myname-cluster-admin-binding --clusterrole=cluster-admin --user=myname@mycompany.com
```

Indenting the second part of the command to graphically show that they are options that belong to the previous line command.